packages/apps/plugins/plugin-thread/src/translations.ts

```diff
@@ -14,7 +14,7 @@ export default [
         'delete thread label': 'Delete thread',
         'create thread label': 'Create thread',
         'message placeholder': 'Reply...',
-        'settings standalone label': '[Experimental] Enable standalone thread creation',
+        'settings standalone label': 'Enable standalone thread creation (experimental).',
```

Comment:
```suggestion
        'settings standalone label': 'Enable standalone thread creation (experimental)',
```
We've deliberately removed all the periods from the settings labels, they look weird in the settings panel and they're not really full sentences.

I think the experimental label going to the end makes sense, but if you're changing that here, could you change all of them so they're consistent?